tests/file_reader_client_fs_notify_spec.lua

```diff
@@ -0,0 +1,60 @@
+require("busted.runner")()
```

Comment:
Is it possible to create files in "tests/data/file_reader_client/*" dynamically during test run in setup phase and removed after test is finished?

Reply:
Now that tests create and remove files dynamically.